### ISSUE TITLE
[0.9.1.9 beta release-136] Migration crash from 9.0.5 -> 9.1.x

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#19890]
Server migration from version 0.9.0.5 into 0.9.1.x causes the following errors which prevents the server from loading

[log_210101051000.log](https://github.com/StrangeLoopGames/EcoIssues/files/5759156/log_210101051000.log)
[migrations.log](https://github.com/StrangeLoopGames/EcoIssues/files/5759157/migrations.log)

 Part of the exception from the logs
```
[17:10:36.875] [23] [Error] [Eco] Failed to load server, Exception was Exception: ArgumentException
Message:An item with the same key has already been added. Key: Vote
Source:System.Private.CoreLib

System.ArgumentException: An item with the same key has already been added. Key: Vote
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at Eco.Shared.Utils.BidirectionalMap`2.Add(TKey key, TVal val)
   at Eco.Stats.ShortNameMap.<Init>b__8_0(ShortNameEntry x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at Eco.Stats.ShortNameMap.Init(List`1 collection)
   at Eco.Stats.StatInfoManager.Initialize(List`1 entries)
   at Eco.Stats.Stats.Initialize(TimedTask timer)
   at Eco.Server.PluginManager.InitializePlugins()
   at Eco.Server.PluginManager.PostCreateAsync(StartupInfo startupInfo)
   at Eco.Server.Startup.StartAsync(StartupInfo startupInfo)
```

Edit : Added save link
https://drive.google.com/file/d/1Vbf1qswW7aGRCTVi2cWrAnKWBQHhKRLU/view?usp=sharing

**Comment by mirasrael** [comment]
Successfully migrated and started on latest staging build. Do we need to fix it in 9.1?

**Comment by mirasrael** [comment]
The issue with LiteDB table name mapping which happened because of non thread-safe mapping creation. 
I fixed root cause: https://github.com/StrangeLoopGames/Eco/pull/7549
But DB file already corrupted. You can use fixed file:  https://drive.google.com/file/d/1tC-afGFwdp9Vay9VhtR5Em9DBcbtXep2/view?usp=sharing

Or just remove DB file.
